File: tests/exercises/echo-function/solution/unknown-return-type.cs
using System;

public struct Coords
{
    public Coords(double x, double y)
    {
        X = x;
        Y = y;
    }

    public double X { get; }
    public double Y { get; }

    public override string ToString() => $"({X}, {Y})";
}

class Submission
{
    public static Coords Echo(string content)
    {
        return new Coords(5.5, 7.5);
    }
}

File: tests/exercises/echo-function/solution/unknown-return-type.cpp
#include <string>

class Coord {
public:
    int x;
    int y;

    Coord(const int& x, const int& y) : x(x), y(y) {}
    friend std::ostream& operator<<(std::ostream& os, const Coord& c)
    {
        return os << "Coord[x=" << c.x << ", y=" << c.y << "]";
    }

};

// Function that returns a Coord object
Coord echo(std::string content) {
    return Coord(5, 7);
}
